src/dmd/lambdacomp.d

```diff
@@ -17,51 +29,81 @@ import dmd.statement;
 import dmd.tokens;
 import dmd.visitor;
 
+/**
+ * This enum is used to report the type of the expression
+ * found when visiting the AST nodes to higher levels of the
+ * AST.
+ */
 enum ExpType
 {
     None,
     EnumDecl,
     Arg
 }
 
+/**
+ * The serialize visitor visits a FuncLiteralDeclaration
+ * AST node, serializing the implementation of the lambda
```

Comment:
What does `serialization of the implementation` mean? Is a string produced? A hash?